docs/LightningNetwork.md

```diff
@@ -44,122 +45,146 @@ export BTCPAYGEN_LIGHTNING="implementationgoeshere"
 
-Go to **Server Settings > Services > Spark Server > See information**
+Go to Server Settings > Services > Spark Server > See information.
 
-![Accessing Spark wallet](./img/SparkConnect.jpg "Accessing Spark wallet")
+### Control your CLN via the command-line: lightning-cli
 
-![Accessing Spark wallet](./img/SparkConnect2.jpg.png "Accessing Spark wallet")
+Similar to `lncli`, CLN can be accessed via the command-line using the shell script `bitcoin-lightning-cli.sh`.
 
-### c-lightning Commands lightning-cli
-
-To use **clightning CLI** it is the same like above for `lncli` but instead you use the shell script `bitcoin-lightning-cli.sh`
 
 If you're on Docker make sure you're in docker directory.
 
 ```bash
 sudo su -
 cd btcpayserver-docker
-./bitcoin-lightning-cli.sh
+./bitcoin-lightning-cli.sh $command
+./bitcoin-lightning-cli.sh getinfo #show info about the node
+```
+Run `./bitcoin-lightning-cli.sh help` to see a full list of commands or check the full [API documentation](https://lightning.readthedocs.io/).
+
+## Lightning node backup
+
+Before you start transacting using your new lightning node, consider backing up the **on-chain** wallet. Steps:
+1. **for LND**: storing a copy of the LND seed.
+Go to "Server Settings" > "Services" > "LND Seed Backup" and select "See information"
+2. **for CLN**: storing a copy of the [hsm_secret](https://lightning.readthedocs.io/BACKUP.html#hsm-secret)
+
+Acknowledge the limitations of **off-chain** payment channel backups and associated risks.
+\
+See [backup FAQ](https://docs.btcpayserver.org/Docker/backup-restore/#lightning-channel-backup) if you are running the BTCPay Server instance with Docker.
+
+## Funding your on-chain wallet
+
+Now that your lightning node is active, before opening lightning payment channels, you will need to fund the on-chain wallet.
+
+The on-chain funding process can be performed in two ways:
+1. via the Ride The Lightning (RTL) UI interface
+- Select a "Store" and go to the "Lightning" section 
+- Under "Services", select "Ride The Lightning"
+- In the RTL app, go to "On-chain", select "Receive" under the "On-chain Transactions" menu
+- Select "Generate Address" and use it as the destination for the allocated funds
+2. via the command-line using `bitcoin-lncli.sh` or `bitcoin-lightning-cli.sh`
+```bash
+sudo su -
+cd btcpayserver-docker
+./bitcoin-lncli.sh newaddress p2wkh #for LND
+./bitcoin-lightning-cli.sh newaddr  #for CLN
+{
+   "address" / "bech32": "bc1..........." #use this as the destination for the allocated funds
+}
 ```
+Once your on-chain lightning node is funded you, it's time to connect to other nodes on the network and open payment channels.
 
-E.g. to list all commands: `./bitcoin-lightning-cli.sh help`
-or show info about the node `./bitcoin-lightning-cli.sh getinfo`
+Check out [Payment channels](./LightningNetwork%20-%20Payment%20channels.md) for recommendations on opening payment channels, liquidity management and more.
```

Comment:
```suggestion
Check out [Payment channels](./LightningNetwork-Payment-channels.md) for recommendations on opening payment channels, liquidity management and more.
```

Because you have spaces in your file name, the above happens, please dont use spaces in file names. 
If anything, use -, but just LNPaymentChannels would do to.

Reply:
ps. you can disregard this change it's just to show how it should be :)